multimedia/welle.io/Portfile

```diff
@@ -0,0 +1,100 @@
+                        with support for rtl-sdr (RTL2832U) and airspy. It supports  \
+                        high DPI and touch displays and it runs even on cheap computers  \
+                        like Raspberry Pi 2/3 and 100€ China Windows 10 tablets.
+
+platforms               darwin
+license                 GPL-2
+license_noconflict      openssl
+
+homepage                https://www.welle.io/
+
+checksums               rmd160  05113c73dc7b97fed256d91cd6a971dabc30c1ce \
+                        sha256  99acaea8386b092ebc80cd6c689ab0106a9f74ad121c04d0ae893f4e35a5a3e0 \
+                        size    1613717
+
+depends_lib-append      port:faad2 \
+                        port:fftw-3-single \
+                        port:lame \
+                        port:libusb \
+                        port:mpg123 \
+                        port:${qt5.name}-qtbase \
+                        port:${qt5.name}-qtcharts \
+                        port:${qt5.name}-qtdeclarative \
+                        port:${qt5.name}-qtmultimedia \
+                        port:${qt5.name}-qtquickcontrols \
+                        port:${qt5.name}-qtquickcontrols2
+#depends_build-append    port:${qt5.name}-qttools
+
+default_variants    +rtlsdr +soapysdr +airspy
+
+variant rtlsdr description {Add RTL-SDR support} {
+    configure.pre_args-append \
+        -DRTLSDR=ON
+    depends_lib-append port:rtl-sdr
+}
+
+variant soapysdr description {Add SoapySDR support} {
+    configure.pre_args-append \
+        -DSOAPYSDR=ON
+    depends_lib-append port:SoapySDR
+}
+
+variant airspy description {Add Airspy support} {
+    configure.pre_args-append \
+        -DAIRSPY=ON
+    depends_lib-append port:airspy
+}
+
+variant morepatches description {Additional patches with bugfixes and features} {
+    patchfiles-append 385-fix_rtl-sdr_setFrequency-v2.patch \
+                    386-fixFavorit.patch \
+                    387-StyleCommandLineArg-remplaceGitCommit-v3.patch \
+                    387-fixQMLforKDE-v3-411Compatible-v2.patch \
+                    400-french-v3.patch \
+                    405-welle.io-2.0-beta3-langComboBox-v2.patch \
+                    406-welle.io-2.0-beta3-mainTr.patch \
+                    407,409-welle.io-2.0-beta3-newTr-v3.patch \
+                    408-welle.io-2.0-beta3-QMLTr.patch \
+                    410,413-diff-EnsembleName.patch \
+                    411-radioView-wordwrapStationName,AlignAntenna,AntennaKdeSupport.patch \
+                    414-Fit_audiomode_in_view.patch \
+                    415-Fit_service_details.patch \
+                    421-stationNameAsWindowTitle.patch \
+                    428-soapyInitArgs-contreFab16.patch \
+                    qqc2-style-gui-v2-ListModel-surFab16.patch \
+                    updateTranslationManagement-requires405,406.patch
+    post-patch {
+        file copy -force ${filespath}/fr_FR.qm ${worksrcpath}/src/welle-gui/i18n/
+    }
+}
+
+cmake.out_of_source yes
```

Comment:
This line is automatically added when using the `cmake 1.1` portgroup and can be removed here.

Reply:
done